src/librustdoc/lib.rs

```diff
@@ -256,6 +256,9 @@ pub fn opts() -> Vec<RustcOptGroup> {
         unstable("sort-modules-by-appearance", |o| {
             o.optflag("", "sort-modules-by-appearance", "sort modules by where they appear in the \
                                                          program, rather than alphabetically")
+        unstable("deny-render-differences", |o| {
```

Comment:
Missing `}),` above this line 😒 

```
[00:43:34] error: incorrect close delimiter: `]`
[00:43:34]    --> /checkout/src/librustdoc/lib.rs:263:5
[00:43:34]     |
[00:43:34] 263 |     ]
[00:43:34]     |     ^
[00:43:34]     |
[00:43:34] note: unclosed delimiter
[00:43:34]    --> /checkout/src/librustdoc/lib.rs:256:52
[00:43:34]     |
[00:43:34] 256 |         unstable("sort-modules-by-appearance", |o| {
[00:43:34]     |                                                    ^
[00:43:34] 
[00:43:34] error: incorrect close delimiter: `]`
[00:43:34]    --> /checkout/src/librustdoc/lib.rs:263:5
[00:43:34]     |
[00:43:34] 263 |     ]
[00:43:34]     |     ^
[00:43:34]     |
[00:43:34] note: unclosed delimiter
[00:43:34]    --> /checkout/src/librustdoc/lib.rs:256:17
[00:43:34]     |
[00:43:34] 256 |         unstable("sort-modules-by-appearance", |o| {
[00:43:34]     |                 ^
[00:43:34] 
[00:43:35] error: expected one of `.`, `;`, `?`, `}`, or an operator, found `unstable`
[00:43:35]    --> /checkout/src/librustdoc/lib.rs:259:9
[00:43:35]     |
[00:43:35] 258 |                                                          program, rather than alphabetically")
[00:43:35]     |                                                                                               - expected one of `.`, `;`, `?`, `}`, or an operator here
[00:43:35] 259 |         unstable("deny-render-differences", |o| {
[00:43:35]     |         ^^^^^^^^ unexpected token
[00:43:35] 
[00:43:35] error: aborting due to 3 previous errors
[00:43:35] 
[00:43:35] error: Could not compile `rustdoc`.
```